setup/.env.sandbox

```diff
@@ -15,6 +15,12 @@ ENTRA_CLIENT_NAME="Otto (Sandbox)"
 # Short abbreviation for the organization to be used in resource names
 ORGANIZATION=JUS
 
+# The IP address range that is allowed to access the application. For multiple IPs, use a comma separator.
+ALLOWED_IPS=199.212.215.11/32
+
+# Flag to enable/disable private network access
+USE_PRIVATE_NETWORK=false
```

Comment:
Why not make this True? Presumably that would just mean requiring a VPN connection for access?

Reply:
For the sandbox and dev, we won't be able to access the resources if we make them private because Cloud, SSC, and IT Sec can't open ports to our specific workstations.